pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -1218,11 +1218,17 @@ func registerClusterToCluster(r registry.Registry) {
 			pdSize:             100,
 			workload:           replicateBulkOps{},
 			timeout:            2 * time.Hour,
```

Comment:
In our ad-hoc runs how close is this timeout? Wondering if we want to add another hour in case fingerprinting is slow.

Reply:
The test runtime was 60 minutes. So I think a doubled timeout is reasonable.